org.eclipse.scout.rt.uinotification/src/main/java/org/eclipse/scout/rt/api/uinotification/UiNotificationPutOptions.java

```diff
@@ -27,6 +35,10 @@ public Long getTimeout() {
     return m_timeout;
   }
 
+  /**
+   * @param transactional
+   *     Whether a notification is processed as a transaction member or not. Default is {@code true}.
```

Comment:
Maybe write more clearly:

Whether the notification should be put into the registry only if the current transaction is committed successfully.